.changeset/lovely-months-train.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nhost/dashboard': minor
+---
+
+fix: dashboard: fix SQL editor "Track this"
```

Comment:
```suggestion
fix: "Track this" option within the SQL editor now correctly updates the metadata
```

this is a suggestion mainly to remove the dashboard mention because this will be part of the dashboard changelog anyways